django/general/templates/general/about.html

```diff
@@ -0,0 +1,10 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block main %}
+
+<section>
+    <h2>About</h2>
```

Comment:
Is this going to be WYSIWYG content, or just coming in a future PR?

Reply:
future PR, this is just setting up pages. it'll be static content (not user managed, as won't be regularly updated, so not necessary).